rest/data/interface.go

```diff
@@ -121,6 +121,7 @@ type Connector interface {
 	// limit, and sort to provide additional control over the results.
 	FindTestsByTaskId(string, string, string, string, int, int) ([]testresult.TestResult, error)
 	FindTestsByTaskIdFilterSortPaginate(string, string, string, string, int, int, int, int) ([]testresult.TestResult, error)
+	FindTaskResultsByVersion(string, string, []string, int, int, int) ([]task.Task, error)
```

Comment:
I would name this FindTasksByVersion to avoid confusion with test results

Reply:
i chose this name to purposefully relate the two. can change tho